docs/copilot/getting-started-chat.md

```diff
@@ -15,140 +15,182 @@ If you're new to using GitHub Copilot in VS Code, see the [GitHub Copilot Gettin
 
-If you want to use GitHub Copilot, you either need a subscription for GitHub Copilot in your personal account, or you need to be assigned a seat by your organization.
+* To use GitHub Copilot in VS Code, you must have the [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) extension. When you install this extension, the [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) extension is also installed.
 
-To get started, go through the steps to [Set up VS Code for GitHub Copilot](/docs/copilot/getting-started.md#set-up-vs-code-for-github-copilot).
+* To use GitHub Copilot, you must have an active subscription for GitHub Copilot in your personal account, or you need to be assigned a seat by your organization.
 
-## Install the GitHub Copilot Chat extension
+Follow these steps to [Set up GitHub Copilot in VS Code](/docs/copilot/getting-started.md#set-up-vs-code-for-github-copilot).
 
-The Chat features are available by installing the [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) extension.
+## Get your first Copilot chat conversation
 
-> <a class="install-extension-btn" href="vscode:extension/GitHub.copilot-chat">Install the GitHub Copilot Chat extension</a>
+GitHub Copilot Chat is a chat interface that lets you interact with GitHub Copilot, to ask and receive answers to coding-related questions. The chat interface provides access to coding information and support without requiring you to navigate documentation or search online forums.
 
-## Get your first Copilot chat conversation
+In this tutorial, you'll be creating a simple Node.js web application. So, let's get started and ask Copilot to tell us more about web frameworks for Node.js.
 
-As you're iterating over your code, there might be moments where you need help from your AI pair programmer to accomplish a task. In the [Copilot Getting Started](/docs/copilot/getting-started.md), you used Copilot inline suggestions and optionally provided context by using code comments.
+1. Open the Chat view from the Activity Bar or press `kb(workbench.action.chat.open)`.
 
-In some cases, it might be more efficient to just ask Copilot about a specific topic by using a chat message. For example, a common cause for errors is when a method receives invalid input. In this step, you start with a simple calculator in TypeScript, and use GitHub Copilot Chat to refactor the code to add error handling.
+    ![Screenshot of VS Code editor, showing the Copilot Chat view, highlighting the chat control in the Activity bar.](./images/getting-started-chat/copilot-chat-view.png)
 
-![Video that shows asking Copilot to add error handling to our code. Copilot presents a diff view with the suggested code changes.](./images/getting-started-chat/copilot-chat-add-error-handling.gif)
+    > **Tip**: Drag the Chat view to the Secondary Side bar to keep both the Chat view and Explorer view open.
```

Comment:
I love this tip (and frequently give it in my sessions), though readers might not know what the secondary sidebar is (and I wonder if they could be confused why doesn't the above image show it). Maybe just linking to a doc on what the secondary sidebar is could help?